cmd/urchin-admin/main.go

```diff
@@ -6,10 +18,16 @@ import (
 	"os"
 
 	_ "github.com/go-sql-driver/mysql"
+	"github.com/rs/zerolog/log"
+	swaggerFiles "github.com/swaggo/files"
+	ginSwagger "github.com/swaggo/gin-swagger"
+
 	admin_app "github.com/matheusgomes28/urchin/admin-app"
 	"github.com/matheusgomes28/urchin/common"
 	"github.com/matheusgomes28/urchin/database"
-	"github.com/rs/zerolog/log"
+
+	// Importa los documentos de Swagger generados para que estén disponibles.
```

Comment:
Maybe I don't understand this, but you import the `docs/admin` to make them available here - but where is it used? I must admit I don't know how `swaggo/swag` works.